backend/src/executors.py

```diff
@@ -7,48 +7,122 @@
 from typing import Sequence, List, Tuple
 from transformers import BertModel, BertTokenizer
 from jina import Executor, requests, Document, DocumentArray
-
 from backend_config import top_k, embeddings_path, results_path, print_logs
-from utils import partition
+
+from utils import generate_path, partition
 
 
 class ProtBertExecutor(Executor):
     """ProtBERT executor: https://huggingface.co/Rostlab/prot_bert"""
 
+    __model = None
+
+    __tokenizer = None
+
     def __init__(self, **kwargs):
         super().__init__()
 
-        tokenizer = BertTokenizer.from_pretrained(
-            "Rostlab/prot_bert", do_lower_case=False
+        self.model = ProtBertExecutor.__model
+        self.tokenizer = ProtBertExecutor.__tokenizer
+
+    @staticmethod
+    def initialize_executor():
+
+        # If the model is not already cached ...
+        if not os.path.isdir("./models/prot_bert"):
```

Comment:
Here the path should be loaded from a config file (the same as for the Dockerfile). For now we have `backend_config.py`.

Reply:
I'll look into the YAML configuration for Jina tonight over how I can store variables there to act as a whole for the backend project.

Thanks for the referral!